runtime/syntax/shared/typescriptcommon.vim

```diff
@@ -623,25 +652,24 @@ syntax keyword typescriptReadonlyArrayKeyword readonly
 
 
 " extension
-if get(g:, 'yats_host_keyword', 1)
-  syntax keyword typescriptGlobal containedin=typescriptIdentifierName Function Boolean
-  " use of nextgroup Suggested by Doug Kearns
+if get(g:, 'typescript_host_keyword', 1)
```

Comment:
I renamed `g:yats_host_keyword` to `g:typescript_host_keyword` because I believe the original variable name is a mistake when this file was imported from yats.vim because it is weird that the original 3rd party plugin's name remains. Please let me know if my assumption is wrong.

Reply:
That is probably fine, since it doesn't seem to be documented :/ and also unused in other runtime scripts, if I see that correctly. Can you please add a short paragraph about this variable in `:h syntax.txt` at a tag `ft-typescript-syntax` (similar to other syntax descriptions).

Reply:
Sure. I'll try to add document.